package.json

```diff
@@ -10,6 +10,10 @@
   "author": "",
```

Comment:
Please add all necessary run common commands like `build`, `development` should be in `package.json` inside `scripts'. 
```json
"scripts": {
    "dev": "webpack --mode development",
    "build": "webpack --mode production",
    "start": "webpack-dev-server --mode development --open" // optional
  },
```